test/constrained-generics/special-method-warnings/user-hash-warning-generic-concrete-impl.future

```diff
@@ -0,0 +1 @@
+secondary method receiver types are unknown and don't flag special method warning
```

Comment:
This doesn't use our normal future formatting.  Typically we start with:
```
<category>: short description
#<issue number>

optional longer description
```
https://chapel-lang.org/docs/latest/developer/bestPractices/TestSystem.html#a-test-that-tracks-a-failure states this and test/functions/ferguson/hijacking/issue-18081.future has an actual example of it.